Dokument/Dateien/Religion.tex

```diff
@@ -158,49 +165,62 @@ \subsection{\textbf{Aina} - Göttin der Lebensfreude und Liebe}
 kurz und der Mittsommer nahe war, eine junge Frau vor seiner Tür stand und ihn zu einem Wettbewerb herausforderte. Gäste wurden geladen, um die Kunstfertigkeit der Kontrahenten 
 zu bewerten und sich ihrer Kunst zu erfreuen. Doch jedes Stück, das Glemric aufspielte, vermochte sein Gast zu überbieten, bis letzlich sogar Glemric selbst einsehen musste, 
 dass er geschlagen war. Noch an diesem Abend gelobte er, seine Ehre wiederherzustellen und sich bei seinem Gast zu revanchieren. Und so geschah es, dass Glemric ein Jahr später 
-in derselben Nacht Aina zu einer Revanche herausforderte. 
+in derselben Nacht Aina zu einer Revanche herausforderte. \\~\\
+positive Gemüter: Glück, Hingabe, Enthusiasmus, Fröhlichkeit, Freiheit, Menschlichkeit, Kunst, Kultur, Charme, Kreativität, Freude
 
-\subsection{\textbf{Faelan} - Gott der Schläue und des Glücks}
+\subsection{\textbf{Faelan} - Gott der Weisheit}
 Weisheit kann viele Dinge bedeuten. Die Fähigkeit, Zusammenhänge zu begreifen und auszunutzen. Die Fähigkeit, den für sich selbst und Andere besten Weg zu erkennen. Und die 
-Fähigkeit, aus dem Wenigen, was man hat, das Beste zu machen. Faelan verkörpert diese Fähigkeiten auf seine Art und Weise. Er hat weder Asdars Stärke noch Erlins Urteilsvermögen - 
-und dennoch ist er regelmäßig in der Lage, die beiden auszutricksen und ihnen dennoch keinen bleibenden Schaden zuzufügen.\\
+Fähigkeit, aus dem Wenigen, was man hat, das Beste zu machen. Faelan verkörpert alle diese Aspekte. Einerseits ist der Gott von Wissen und 
+Gelehrsamkeit, der die Menschen Schrift, Wissen und Magie lehrt. Andererseits ist - beziehungsweise war - Faelan ein Gott der Schläue und 
+der günstigen Gelegenheiten, der den Menschen überlegtes und zielstrebiges Handeln beibringt. Diese zweite Seite von Faelans Wesen ist mit 
+dem Erstarken des Klerus immer mehrin Vergessenheit geraten, da die Verehrung eines Gottes der Diebe und Reisenden wenig erwünscht war.\\
```

Comment:
des Klerus immer mehrin Vergessenheit --> des Klerus immer mehr in Vergessenheit